server/controllers/userController.js

```diff
@@ -1,12 +1,27 @@
-const User = require('../models/User');
-
-async function getUser(req, res) {
-  const user = await User.findById(req.params.userId).select('-password');
-
-  if (!user) {
-    return res.status(404);
-  };
-  res.status(200).send(user);
-}
-
-module.exports = { getUser };
+const User = require('../models/User');
+
+async function getUser(req, res) {
+  const user = await User.findById(req.params.userId).select('-password');
+
+  if (!user) {
+    return res.status(404);
+  }
+  res.status(200).send(user);
+}
+
+async function updateUser(req, res) {
+  const id = req.params.userId;

```

Comment:
Security issue: you need to check if this req.params.userId is the same as the authenticated user in the middleware

Reply:
Got it! Can't have any user updating any other user.